__tests__/app.test.js

```diff
@@ -43,3 +43,51 @@ describe(" GET /api/topics ", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET - status:200, responds with an article array of object", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+  test("GET - status 200, responds with a sorted order value ", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at")
```

Comment:
Don't need to test with a query - queries come in at ticket 10. This test is fine if you ".get("/api/articles") - without the query.